src/e2e-test/features/gcsdelete/GCSDelete.feature

```diff
@@ -55,6 +55,63 @@ Feature: GCS Delete - Verification of GCS Delete plugin
     Then Verify the pipeline status is "Succeeded"
     Then Verify multiple objects "gcsDeleteObjectsList" deleted successfully by GCS Delete action plugin
 
+  @GCS_WILDCARD_TEST
```

Comment:
nit, would it be better to call it GCS_DELETE_WILDCARD_TEST ?

Reply:
ok